contracts/ColonyNetworkMining.sol

```diff
@@ -85,8 +85,9 @@ contract ColonyNetworkMining is ColonyNetworkStorage {
     address clnyToken = IMetaColony(metaColony).getToken();
     require(clnyToken != address(0x0), "colony-reputation-mining-clny-token-invalid-address");
 
-    inactiveReputationMiningCycle = new EtherRouter();
-    EtherRouter(inactiveReputationMiningCycle).setResolver(miningCycleResolver);
+    EtherRouter e = new EtherRouter();
```

Comment:
Because implicit conversion from a contract to address type is not supported in solc 0.5.0. Additionally the cast from non-payable address to a payable contract type (such as EtherRouter) is also not supported. Additionally the new single line replacement is much cleaner in its purpose.

Reply:
👍